exercises/diffie-hellman/example.js

```diff
@@ -0,0 +1,48 @@
+
+var DiffieHellman = function(p, g){
+  this.p = p;
```

Comment:
I'm not sure `p` and `g` should be attributes of `DiffieHellman`. The solution might work without them being attributes, just parameters to the function.